requirements.txt

```diff
@@ -1,4 +1,7 @@
-spacy
+# spacy
+spacy @ git+https://github.com/rmitsch/spaCy.git@feature/candidate-generation-by-docs
```

Comment:
Experimental branch for entity batching by documents (not leveraged by the current implementation, but might be soon).

Reply:
Agreed, same for the `sqlite-spellfix` branch. I submitted a PR to the repo, but I don't know if it's still being maintained. We could also vendor that at some point, it's pretty much just the build setup for the SQLite extension for fuzzy search.